1-exercises/I-string-replace/exercise.js

```diff
@@ -14,6 +14,12 @@ let story =
   "I like dogs. One day I went to the park and I saw 10 dogs. It was a great day.";
 
 let result = story.replace("", "");
+const replacements = {
+  "dogs": "cats",
+  "day": "nights",
```

Comment:
Good starting point, like the thought process here but the implimentation needs work